.prettierrc

```diff
@@ -1,12 +0,0 @@
-{
-  "semi": false,
-  "arrowParens": "avoid",
-  "insertPragma": false,
-  "printWidth": 80,
-  "proseWrap": "preserve",
-  "singleQuote": true,
-  "trailingComma": "none",
-  "useTabs": false,
-  "jsxBracketSameLine": false,
-  "bracketSpacing": false
-}
```

Comment:
I'd not remove this. I guess pretty much everyone is using prettier to format the .vue files.